.github/workflows/docs-ci.yml

```diff
@@ -1,39 +1,55 @@
-# This workflow will build and deploy docs to GitHub Pages whenever something is pushed to the default branch
+name: Build docs
 
-name: Docs
-
-on: push # but only to default branch, see if below
+on:
+  push:
+    branches: [ "0.9.x" ]
+  tag:
+  workflow_dispatch:
 
+# Gives the workflow permissions to clone the repo and create a page deployment
 permissions:
-  contents: write
+  id-token: write
+  pages: write
 
+# This workflow updates the versioned docs on dokka.kord.dev using the following strategy
+# 1) pull cache to docs-cache
+# 2) Call dokkaHtmlMultiModule for current version
+#   - if this is a release build move current docs into docs-cache/$version
+#   - otherwise move it into docs-cache/current
+# 3) Build GitHub Pages artifact
+#   - Move all folders not named "current" into docs/<folder>
+#   - Move current into docs/
```

Comment:
i don't get where this is done

Reply:
the produceDocs tasks

Reply:
is it custom? cause if so, you didn't commit it